nemo/collections/tts/parts/transformer/conv_transpose.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2020 NVIDIA. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from torch import nn
+
+
+class Conv(nn.Module):
```

Comment:
Do we really need this class? Are there no equivalent classes anywhere else in the repo?

Reply:
Conv class added transposes for input tensor. Seems like this is need for implementing DurationPredictor as sequence of modules.
Not sure about other such things in NeMo.

Reply:
Did you remove this file?